posts/2023-04-13-Updating-musl-targets.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Updating Rust's Linux musl targets"
+author: Wesley Wiser
+description: "musl targets will soon ship with musl 1.2"
+team: The Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Beginning with Rust 1.nn (slated for TODO), the various `*-linux-musl` targets will [ship][PR] with musl 1.2.3.
```

Comment:
Calling this out so I don't forget to update it when we know which release will have the change.